src/Tribe/Promoter/Observer.php

```diff
@@ -137,14 +157,84 @@ public function ticket_moved_type( $ticket_type_id, $destination_id, $source_id,
 		}
 	}
 
+	/**
+	 * Observer when an attendee is moved from a post to another and notify Promoter about changes on both events
+	 *
+	 * @since TBD
+	 *
+	 * @param int $ticket_id the ticket which has been moved
+	 * @param int $source_ticket_type_id the ticket type it belonged to originally
+	 * @param int $target_ticket_type_id the ticket type it now belongs to
+	 * @param int $source_event_id the event/post which the ticket originally belonged to
+	 * @param int $target_event_id the event/post which the ticket now belongs to
+	 * @param int $instigator_id the user who initiated the change
+	 *
+	 * @return void Action hook with no return.
+	 */
+	public function ticket_moved( $ticket_id, $source_ticket_type_id, $target_ticket_type_id, $source_event_id, $target_event_id, $instigator_id ) {
+		$this->notify( $source_event_id );
+		// Prevent to send the same response twice if the ID's are the same.
+		if ( $source_event_id !== $target_event_id ) {
+			$this->notify( $target_event_id );
+		}
+	}
+
+	/**
+	 * Action fired when an PayPal attendee ticket is updated.
+	 *
+	 * @since TBD
+	 *
+	 * @param int    $attendee_id           Attendee post ID.
+	 * @param string $order_id              PayPal Order ID.
+	 * @param int    $product_id            PayPal ticket post ID.
+	 * @param int    $order_attendee_id     Attendee number in submitted order.
+	 * @param string $attendee_order_status The order status for the attendee.
+	 *
+	 * @return void
+	 */
+	public function tpp_attendee_updated( $attendee_id, $order_id, $product_id, $order_attendee_id, $attendee_order_status ) {
+		$this->notify_ticket_event( $product_id );
+	}
+
+	/**
+	 * Action fired when a PayPal has had attendee tickets generated for it.
+	 *
+	 * @since TBD
+	 *
+	 * @param int    $product_id PayPal ticket post ID.
+	 * @param string $order_id   ID of the PayPal order.
+	 * @param int    $qty        Quantity ordered.
+	 *
+	 * @return void
+	 */
+	public function tpp_tickets_generated_for_product( $product_id, $order_id, $qty ) {
+		$this->notify_ticket_event( $product_id );
+	}
+
 	/**
 	 * Notify the connector of changes when the event was deleted
 	 *
 	 * @since 4.10.9
 	 *
 	 * @param $post_id
+	 *
+	 * @deprecated TBD
+	 *
 	 */
 	public function delete_post( $post_id ) {
+		_deprecated_function( __METHOD__, '4.11.5', __CLASS__ . "::on_event_deleted" );
+
+		$this->on_event_deleted( $post_id );
+	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$post_id" missing

<sub>hash: 39599bd8638f15237486edd0cb85da5a</sub>